packages/permissions/src/hooks/use-is-authorized/use-is-authorized.spec.tsx

```diff
@@ -375,10 +375,56 @@ describe('action rights', () => {
 describe('data fences', () => {
   describe('when general permission is matched', () => {
     describe('when data fence is matched', () => {
-      it('should indicate as authorized', () => {
+      describe('when not overwritten', () => {
+        it('should indicate as authorized', () => {
+          const rendered = render({
+            shouldMatchSomePermissions: true,
+            allAppliedPermissions: [{ name: 'canManageOrders', value: true }],
+            allAppliedDataFences: [
+              {
+                __typename: 'StoreDataFence',
+                type: 'store',
+                group: 'orders',
+                name: 'canViewOrders',
+                value: 'store-1',
+              },
+            ],
+            demandedPermissions: ['ManageOrders'],
+            demandedDataFences: [
+              {
+                type: 'store',
+                group: 'orders',
+                name: 'ViewOrders',
+              },
+            ],
+            selectDataFenceData: ({ type }) => {
+              switch (type) {
+                case 'store':
+                  return ['store-1'];
+                default:
+                  return null;
+              }
+            },
+          });
+
+          expect(
+            rendered.queryByText('Is authorized: Yes')
```

Comment:
now this is where things become confusing.
IMO data-fence is overwritten here, since the general permission is `manage` which matches the demanded.

additionally in this specific test case, when indicating `is authorized`, there is no indication if what we authorized is the a match of `demanded=manage, applied=manage` or `demanded=view, applied=view`

--- 

our goal is to update the convention. "override" is now amended to take `action-right` in account